src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -636,14 +636,12 @@ def _job_task(self, task: Task, remote_wheels: list[str]) -> jobs.Task:
     def _job_dashboard_task(self, jobs_task: jobs.Task, task: Task) -> jobs.Task:
         assert task.dashboard is not None
         dashboard_id = self._install_state.dashboards[task.dashboard]
-        return replace(
-            jobs_task,
-            job_cluster_key=None,
-            sql_task=jobs.SqlTask(
-                warehouse_id=self._warehouse_id,
-                dashboard=jobs.SqlTaskDashboard(dashboard_id=dashboard_id),
-            ),
+        dashboard_url = f"{self._ws.config.host}/sql/dashboardsv3/{dashboard_id}"
```

Comment:
Linking a dashboard via a notebook now. SqlTask does not work with Lakeview dashboard

Reply:
This is outdated, not linking a dashboard in a job anymore as this is not possible with Lakeview